### DISCUSSION TITLE
Seeed Studio XIAO Expansion board | Seeed Studio Wiki

### DISCUSSION THREAD

**Discussion by giscus** [Seeed-Studio/wiki-documents#409] [Q&A]
# Seeed Studio XIAO Expansion board | Seeed Studio Wiki

Seeed Studio XIAO Expansion board

https://wiki.seeedstudio.com/Seeeduino-XIAO-Expansion-Board/

**Comment by TobiasReich** [comment]
Hello,

I recently bought the XIAO Expansion board. The documentation states the option for battery support.
However, when I connected an external battery I noticed the board getting really hot (at the area below the ESP32 - right at the PMIC). I immediately unplugged the battery again. However I seem to have broken something and it doesn't connect to my ESP32 anymore.
Could anyone help me in this regard? What happened? After all the documentation states it is possible to connect a battery there. (I picked the right spot at the LiPo Battery plug).

Was that wrong? I hope it is not permanently broken.
I'd be happy to hear any opinion on this.
